app/src/main/java/com/bucketlist/MainActivity.kt

```diff
@@ -5,6 +5,7 @@ import android.os.Bundle
 import android.widget.ArrayAdapter
 import android.widget.Spinner
 import com.bucketlist.ui.main.MainFragment
+import kotlinx.android.synthetic.main.main_activity.*
```

Comment:
Needed for synthetics

Reply:
Good.  Much less typing with synthetics.  :)